extensions/azdata/src/test/azdataReleaseInfo.test.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as should from 'should';
+import * as sinon from 'sinon';
+import { HttpClient } from '../common/httpClient';
+import { getPlatformReleaseVersion, getPlatformDownloadLink } from '../azdataReleaseInfo';
+
+const emptyReleaseJson = {
+	win32: {},
+	darwin: {},
+	linux: {}
+};
+
+describe('azdataReleaseInfo', function (): void {
+	afterEach(function (): void {
+		sinon.restore();
+	});
+
+	it('getAzdataReleaseInfo throws with invalid JSON', async function (): Promise<void> {
+		sinon.stub(HttpClient, 'getTextContent').resolves('invalid JSON');
+		await should(getPlatformReleaseVersion()).be.rejected();
```

Comment:
Should this be calling getAzdataReleaseInfo?

Reply:
getAzdataReleaseInfo is the function that is actually throwing here so it's technically correct but that's an implementation detail so I'll rename it